websites/V/Vukkybox/dist/metadata.json

```
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.5",
+    "service": "Vukkybox",
+    "author": {
+        "name": "Mitchellonii",
+        "id": "490664749315653642"
+    },
+    "category": "games",
+    "logo": "https://imgur.com/dIpE7wX",
+    "thumbnail": "https://imgur.com/uS5AFMi",
```

these are not image links
```suggestion
    "logo": "https://i.imgur.com/dIpE7wX.png",
    "thumbnail": "https://i.imgur.com/uS5AFMi.png",
```